packages/accordion-react/src/Accordion.tsx

```diff
@@ -1,26 +1,33 @@
 import React, { ReactNode, useState } from "react";
+// @ts-ignore
```

Comment:
Maybe a comment to say why, we should probably not encourage ts-ignore all willy nilly (https://www.merriam-webster.com/dictionary/willy-nilly)

Reply:
And we should consider adding types to the upstream NRK project if we keep using this.

Reply:
Agreed. Added comment about this